workshop.json

```diff
@@ -8,7 +8,7 @@
         {
             "name": "default",
             "requirements": [
-                "bzip2_src", "bison_src", "gettext_src", "flex_src", "tools_src"
+                "bzip2_src", "bison_src", "gettext_src", "flex_src", "tools_src", "libtraceevent_src", "libtracefs_src", "tracecmd_src", "bcc", "rt-trace-bpf"
```

Comment:
Is the bpf/bcc stuff relevant to this?

Reply:
probably not,  I will remove for a later PR